lib/battleship.rb

```diff
@@ -0,0 +1,31 @@
+require "./lib/ship"
+require "./lib/board"
+require "./lib/player"
+require "./lib/game"
+
+class Battleship
+ #attr_reader
+  def initialize
+    @auto_cruiser = Ship.new("Cruiser", 3)
```

Comment:
Human speak for my brain:
First - This creates ships for computer and player
Second - This creates boards with generated cells for computer and player
Third - This assigns the ships and boards to computer and player
Fourth - This adds ships to the computer and players "ship cache"
Fifth - This begins the game

LOVE THIS. I can see if we had time to work through iteration 4, we could have adapted this file. Love it.